package/MDAnalysis/core/universe.py

```diff
@@ -387,11 +387,11 @@ def load_new(self, filename, format=None, in_memory=False, **kwargs):
                                  trj_n_atoms=self.trajectory.n_atoms))
 
         if in_memory:
-            self.transfer_to_memory(kwargs.get("in_memory_frame_interval", 1))
+            self.transfer_to_memory(step=kwargs.get("in_memory_frame_interval", 1))
```

Comment:
The `in_memory_frame_interval` of `Universe` should become `in_memory_step`. This means changing the key you are looking for in that line we are commenting, but also change the docstring of `Universe`, and change the kwargs in a handful of lines in the tests (the good things is that the lines in questions will be in the same file you are going to add tests in anyway).